Dockerfile

```diff
@@ -14,8 +14,8 @@ RUN mvn clean package -DskipTests
 FROM eclipse-temurin:21-jre-alpine
 
 WORKDIR /home
-COPY --from=builder /home/target/jpo-sdw-depositor-1.6.0-SNAPSHOT.jar /home
```

Comment:
Learnt from the s3 depositor that we could build with assembly:single descriptor, maybe we could do the same here to avoid constantly updating the version number?

Reply:
This seems like it may be a bit more involved than I initially expected. I have opened a CDOT work item for this with the following description:
> Each time the version of the project is updated, the Dockerfile needs to be updated to reference the new JAR since the version is included in the filename. If the version was not included in the JAR filename, the process of updating would be simplified. The jpo-s3-deposit project can be used a reference for implementing this change.